website_media_size/static/src/css/website_media_size.css

```diff
@@ -0,0 +1,12 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+.o_existing_attachment_cell .o_existing_attachment_size {
+    position: absolute;
+    padding: 3px;
+    background-color: rgba(0, 0, 0, 0.8);
```

Comment:
You can use "opacity"

Reply:
I need the background to be black.

Reply:
... Only if the foreground is white :stuck_out_tongue_winking_eye:

Reply:
A black background will create contrast so it will be easy to read. Opacity will also apply it on the text making it less readable in all cases.